content/docs/guides/datadog.md

```diff
@@ -0,0 +1,114 @@
+---
+title: The Neon Datadog integration
+subtitle: Send metrics and events from Neon Postgres to Datadog
+enableTableOfContents: true
+---
+
+<InfoBlock>
+<DocsList title="What you will learn:">
+<p>How to set up the integration</p>
+<p>The full list of externally-available metrics</p>
+</DocsList>
+
+<DocsList title="External docs" theme="docs">
+<a href="https://docs.datadoghq.com/account_management/api-app-keys/">Datadog API and Application Keys</a>
+</DocsList>
+</InfoBlock>
+
+The Neon Datadog integration lets you monitor Neon database performance, resource utilization, and system health directly from Datadog's observability platform.
+
+<ComingSoon/>
+
+## How it works
+
+The integration leverages a [list of metrics](#available-metrics) that Neon makes available for export to third-party services. By configuring the integration with your Datadog API key, Neon automatically sends metrics from your project to your selected Datadog site. Some of the key metrics include:
+
+- **Connection counts** &#8212; Tracks active and idle database connections.
+- **Database size** &#8212; Monitors total size of all databases in bytes.
+- **Replication delay** &#8212; Measures replication lag in bytes and seconds.
+- **Compute metrics** &#8212; Includes CPU and memory usage statistics for your compute.
+
+## Match the Datadog site to your project region
+
+When you configure the integration, you'll be asked to choose your Datadog site. Choosing a site in the same region as your Neon project helps reduce latency and meet compliance requirements, if those are applicable.
+
+## Prerequisites
+
+Before getting started, ensure the following:
+
+- You have a Neon account and project. If not, see [Sign up for a Neon account](/docs/get-started-with-neon/signing-up).
+- You have a Datadog account, your Datadog agent is set up, and you have the appropriate permissions to create an API key.
+
+## Steps to integrate Datadog with Neon
+
+1. In the Neon Console, navigate to the **Integrations** page in your Neon project.
+2. Locate the **Datadog** card and click **Add**.
+3. Enter your **Datadog API key**. You can generate or retrieve these from retrieve from [API Keys](https://app.datadoghq.com/organization-settings/api-keys) your Datadog organization.
+4. Select the **Datadog site** that best corresponds to your Neon project's region.
```

Comment:
Datadog site is something that users select when creating their account. I don't think they can choose a different region on per-project basis. They will need to create a separate Datadog account if they want data to be hosted elsewhere.

<img width="543" alt="image" src="https://github.com/user-attachments/assets/eaf14bcc-e809-4a6d-a0d1-4e6bdf5e8670">

Reply:
Ok, got it. Looks like a roundabout way to check your region in datadog: compare your URL against a table in their docs. 
https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site

I'll revise to match this.